.vscode/settings.json

```diff
@@ -9,5 +9,5 @@
   "grunt.autoDetect": "off",
   "javascript.validate.enable": false,
```

Comment:
should `javascript.validate.enable` also be turned to true?

Reply:
No, as all .js files have Flow and TS doesn't like that.